source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -41,9 +41,50 @@ namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace GrpcJsonTranscoder {
-
 namespace {
 
+// TODO(alyssawilk) see if we can get proto to expose this code.
```

Comment:
FWIW I thought I could get this via descriptor reflection but it looks like the error stuff is C++ and so rather than Code_descriptor() there's a hard-coded string function:

https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/stubs/status.cc